src/azure-cli/azure/cli/command_modules/monitor/commands.py

```diff
@@ -199,3 +211,20 @@ def load_command_table(self, _):
         g.custom_command('list', 'list_metric_alerts', custom_command_type=alert_custom)
         g.command('show', 'get')
         g.generic_update_command('update', custom_func_name='update_metric_alert', custom_func_type=alert_custom)
+
+    with self.command_group('monitor log-analytics workspace', log_analytics_workspace_sdk, custom_command_type=log_analytics_workspace_custom, client_factory=cf_log_analytics_workspace) as g:
```

Comment:
You already specify client factory through sdk, why do you still specify client factory parameter again?

Reply:
Fix